python/chronos/src/bigdl/chronos/forecaster/utils.py

```diff
@@ -63,8 +70,10 @@ def xshard_to_np(shard, mode="fit", expand_dim=None):
         return yhat
 
 
-def np_to_xshard(x, prefix="x"):
+def np_dataloader_to_xshard(x, prefix="x"):
     from bigdl.orca.data import XShards
+    if isinstance(x, DataLoader):
+        x = x.dataset.tensors[0].numpy()
```

Comment:
currently I think we can leave this dataloader's support for distributed forecaster in next PR